### DISCUSSION TITLE
Why are some tugs "noisier" than others?

### DISCUSSION THREAD

**Discussion by scottveirs** [orcasound/orca-shipnoise#8] [Q&A]
Let's take notes on tug noise heard through the Orcasound live hydrophones and flag times when tugs likely polluted the acoustic environment of SRKWs. Then we can discuss why some seem to radiate noise continuously across a broad band of frequencies while others sound anomalous or cross paths with orcas more often -- and may therefore be noise pollution outliers.

Over time, maybe we can list out the tugs that sound anomalous or happen to overlap with the endangered orcas more than others. Then we can map their MMSIs to the local companies and POCs, and reach out to them. That could set the stage for approaching them as shipnoise end-users, ultimately to learn together why their tugs are noisy or making noise near orcas, and discuss how to quiet them over time.

**Comment by scottveirs** [comment]
Today checking on the hydrophone streams, I noticed two anomalous tug noise events. The first was at Bush Point with an unusual amount of blade rate cavitation. The second was at Sunset Point with a weird fluttering noise over the normal continuous broadband noise I've come to expect from tugs. 

Both tugs seem to be local, with tracks out of Everett today. So, maybe their owners/captains may be more amenable to understanding their noise impact and mitigating it over time, especially when SRKWs are along their route? ...

**Comment by scottveirs** [comment]
# Hustler II (MMSI:316005607) 
## 12/05/24, 10:32:28 AM:  northbound at Bush Point

Here's the AIS track of the Hustler II (via vesselfinder.com) -- 
<img width="1792" alt="Screenshot 2024-12-05 at 10 37 52 AM" src="https://github.com/user-attachments/assets/f7262460-eb30-465e-921f-ba8d2ad112a3">

And [here's what it sounded like on the Bush Point livestream at the time...](https://live.orcasound.net/reports/cand_02yRT9S6QgiUD1jVX8SVcF)

Here's a photo of its dual props from marinetraffic.com --
<img width="1792" alt="Screenshot 2024-12-05 at 10 37 25 AM" src="https://github.com/user-attachments/assets/2a321f37-4a2b-40ad-913a-4cbf9a402571">

**Comment by scottveirs** [comment]
# Glen Cove (MMSI: 366993160)

## 12/5/24: outbound at Sunset Bay

Here is its track --
<img width="1792" alt="Screenshot 2024-12-05 at 10 40 56 AM" src="https://github.com/user-attachments/assets/5d3589b9-4977-4ce3-bf9f-a2f6ca7d8bb9">

Here's what it sounded like --
[https://live.orcasound.net/reports/cand_02yRTEWgRLeNdSmehKoRbu](https://live.orcasound.net/reports/cand_02yRTEWgRLeNdSmehKoRbu)

And here it is routing through the Port Townsend canal. (I wonder if they do this to save fuel when there's a current in Admiralty, or if they are just routing shortest-path to Port Hadlock or something...)

**Comment by dkerlee** [comment]
I'd like to add there are several types of underwater propulsion systems tugs use.

1. **conventional**: almost always two propellors with rudders aft of the props. Occasionally there are also "flanking rudders" where there are rudders in front of the props that make the boat more maneuverable when operating astern. [wiki](https://en.wikipedia.org/wiki/Propeller)
2. **Voith-schneider**: vertical propeller egg-beater type of propulsion. WAY different that a propellor. Very maneuverable, reliable. [wiki](https://en.wikipedia.org/wiki/Voith_Schneider_Propeller)
3. **ASD** - azimuthing stern drive, currently the most popular type. PNW and other west coast ports (I can't speak for east coast or foreign) should be seeing an increase in these types of tugs. [wiki](https://en.wikipedia.org/wiki/Azimuth_thruster) Side note on this wiki article here, is that it has ASD that are open. The tugs I know are 100% kort-nozzeled type.
